### PR TITLE
zapier convert: Generate basic test code for a trigger/create/search

### PR DESCRIPTION
**Pull request by eliangcs**

### CODE REVIEW

File: scaffold/convert/create-test.template.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
+require('should');
+
+const zapier = require('zapier-platform-core');
+
+const App = require('../../index');
+const appTester = zapier.createAppTester(App);
+
+const utils = require('../utils');
```
Comment (BrunoBernardino) [inline scaffold/convert/create-test.template.js]: I don't believe this is necessary.

File: scaffold/convert/create-test.template.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+describe('Creates - <%= KEY %>', () => {
+  zapier.tools.env.inject();
+
+  it('should create an object', (done) => {
+    const bundle = {
+      authData: utils.getAuthData(),
```
Comment (BrunoBernardino) [inline scaffold/convert/create-test.template.js]: I'd try to create some method above (that would differ depending on the app's auth type) and call it here instead.

File: scaffold/convert/search-test.template.js [hunk 3]
```diff
@@ -0,0 +1,28 @@
+require('should');
+
+const zapier = require('zapier-platform-core');
+
+const App = require('../../index');
+const appTester = zapier.createAppTester(App);
+
+const utils = require('../utils');
```
Comment (BrunoBernardino) [inline scaffold/convert/search-test.template.js]: Same as above.

File: scaffold/convert/search-test.template.js [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+describe('Searches - <%= KEY %>', () => {
+  zapier.tools.env.inject();
+
+  it('should get an object', (done) => {
+    const bundle = {
+      authData: utils.getAuthData(),
```
Comment (BrunoBernardino) [inline scaffold/convert/search-test.template.js]: Same as above.

File: scaffold/convert/create-test.template.js [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+      inputData: {
+        name: 'Testing',
+      },
+    };
+
+    appTester(App.creates.<%= KEY %>.operation.perform, bundle)
```
Comment (BrunoBernardino) [inline scaffold/convert/create-test.template.js]: Is there a possibility of `KEY` having a `-`? If so, we may prefer doing `['<%= KEY %>']` here.

File: scaffold/convert/search-test.template.js [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+    };
+
+    appTester(App.searches.<%= KEY %>.operation.perform, bundle)
```
Comment (BrunoBernardino) [inline scaffold/convert/search-test.template.js]: Same as above.

File: scaffold/convert/trigger-test.template.js [hunk 7]
```diff
@@ -0,0 +1,26 @@
+require('should');
+
+const zapier = require('zapier-platform-core');
+
+const App = require('../../index');
+const appTester = zapier.createAppTester(App);
+
+const utils = require('../utils');
```
Comment (BrunoBernardino) [inline scaffold/convert/trigger-test.template.js]: Same as above.

File: scaffold/convert/trigger-test.template.js [hunk 8]
```diff
@@ -0,0 +1,26 @@
...
+
+describe('Triggers - <%= KEY %>', () => {
+  zapier.tools.env.inject();
+
+  it('should get an array', (done) => {
+    const bundle = {
+      authData: utils.getAuthData(),
```
Comment (BrunoBernardino) [inline scaffold/convert/trigger-test.template.js]: Same as above.

File: scaffold/convert/trigger-test.template.js [hunk 9]
```diff
@@ -0,0 +1,26 @@
...
+    };
+
+    appTester(App.triggers.<%= KEY %>.operation.perform, bundle)
```
Comment (BrunoBernardino) [inline scaffold/convert/trigger-test.template.js]: Same as above.

File: scaffold/convert/create-test.template.js (hunk 5 above)
Comment (eliangcs) [reply scaffold/convert/create-test.template.js]: Good catch. Fixed in 0f46d2b.

File: scaffold/convert/create-test.template.js (hunk 2 above)
Comment (eliangcs) [reply scaffold/convert/create-test.template.js]: Fixed all the `authData` issues in bf3c107.

**Comment by eliangcs** [comment]
@BrunoBernardino thanks for the review! I have made some fixes based on your feedback. Have a look when you got a chance. Thanks!

**Pull request merged**